package.json

```diff
@@ -167,7 +167,7 @@
     "mixpanel": "^0.10.1",
     "mkdirp": "^0.5.1",
     "moment": "^2.24.0",
-    "moment-duration-format": "~2.2.2",
+    "moment-duration-format": "^2.2.2",
```

Comment:
I think that 2.3 broke the typings but for sure we now don't have any reason to not change it to a `^`.
After all, whoever tries to change the shrinkwrap will see whether it breaks the build.

Reply:
I didn't know about the typings, but I knew about 2 issues that I raised myself in the upstream repo:
* [ReferenceError: window is not defined in v2.3.0](https://github.com/jsmreese/moment-duration-format/issues/131)
* [ReferenceError: window is not defined in v2.3.1](https://github.com/jsmreese/moment-duration-format/issues/133)

... which had broken balena CLI and required us to pin it to `~2.2.2`. Upstream claims to have fixed it but I have not tested it myself. I was careful to add `npm-shrinkwrap.json` in a separate previous commit to the unpinning, so the CLI should still use `moment-duration-format` v2.2.2 anyway.